docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing a Lens visualization into a space, ensure the visualization's data is present. Without it, the Lens visualization won't work.
+
+[float]
+[[cases-export]]
+=== Export a case
+Exporting cases can help you back up your cases or move them from one space to another. When you export a case, the case details, user actions, and comments are exported to a delimited JSON (ndjson) file. During the export process, text string comments, case alerts, and lens visualizations are exported.
+
+NOTE: Exported alerts will only contain indices' names, the alert document ID, and the ID and name of the rule that triggered the alert. Lens visualizations are exported as a JSON blob.
+
+To export a case:
+
+. Open the main menu, then click *Stack Management > Kibana > Saved Objects*.
+. Search for the case by choosing a saved object type or by entering the case title in the search bar.
+. Select one or more case and click the *Export* button.
+. Click *Export*. A toast message verifying that your file is downloading in the background displays.
+
++
+TIP: Keep the *Include related objects* option enabled to ensure connectors are also exported.
```

Comment:
Actually user actions and comments are always exported because we do that manually (since they're inbound references). The `Include related objects` only controls outbound references.

Reply:
@cnasikas @jonathan-buttner would you mind elaborating on what outbound references for cases and or connectors are?

Reply:
Yeah sure. I think you can leave the tip as you've written it but I'll explain what inbound and outbound references are:

Saved objects can reference other saved objects by design and the `Include related objects` controls whether those referenced saved objects are included in the export. It only applies for references coming from the originating saved object though. So for example let's say you're exporting a case called "Jon's Case", behind the scenes that's actually composed of multiple saved objects but the originating saved object is called a "case" (there are also, user actions, comments, and some others).

If a connector is configured for a case, the case saved object will have a reference to a connector saved object. This is called an outbound reference from the case saved object (and an inbound reference for the connector saved object). The user action and comment saved objects have references to the case they are associated but the reference isn't stored in the case saved object it's stored in the user action and comment themselves.

When the `Include related objects` toggle is enabled the saved object management functionality examines the originating saved object and also exports all the references that are explicitly defined (aka the outbound references). For cases the only outbound reference is the connector. Within cases we extended that functionality so that when a case is exported we also grab the inbound references so that we export the entire case (the case, user actions, comments, etc).

Reply:
Thanks for this @jonathan-buttner! I left the note as it was since inbound references are always exported during the export process.